ci-droid-tasks-consumer-services/src/main/java/com/societegenerale/cidroid/tasks/consumer/services/eventhandlers/BestPracticeNotifierHandler.java

```diff
@@ -125,4 +161,16 @@ private Map findConfiguredPatternsThatMatch(List<PullRequestFile> filesInPr) {
         return matchingPatternsByPullRequestFile;
 
     }
+
+    private void notifyComments(PullRequestEvent event, StringBuilder bestPracticesWarnings) {
+        PullRequest pr = remoteGitHub.fetchPullRequestDetails(event.getRepository().getFullName(), event.getPrNumber());
+        Map<String, Object> additionalInfosForNotification = new HashMap();
+        additionalInfosForNotification.put(PULL_REQUEST, pr);
+
+        notifiers.stream().forEach(n -> n.notify(new User(), new Message(bestPracticesWarnings.toString()), additionalInfosForNotification));
+    }
+
+    private boolean commetExists(StringBuilder warnings) {
```

Comment:
typo in method name -> comme**n**tExists

Reply:
corrected.